demo

```diff
@@ -43,35 +36,76 @@ execute() {
   ${DRY_RUN:-false} || "$@"
 }
 
+#mask oc and tkn to use the $NAMESPACE
+OC() {
+  oc -n ${NAMESPACE} "$@"
+}
+
+TKN() {
+ tkn -n ${NAMESPACE} "$@"
+}
+
+demo.validate() {
+  tkn version >/dev/null 2>&1 || err 1 "no tkn binary found"
+  oc version >/dev/null 2>&1 || err 1 "no oc binary found"
+  return 0
+}
+
 demo.prep() {
-  oc get ns -o name | grep -q namespace/pipelines-tutorial || {
-    oc new-project pipelines-tutorial
+  demo.validate
+  oc get ns "$NAMESPACE" 2>/dev/null  || {
+    oc new-project $NAMESPACE
   }
-  oc create serviceaccount pipeline
-  oc adm policy add-scc-to-user privileged -z pipeline
-  oc adm policy add-role-to-user edit -z pipeline
+
+  OC get sa pipeline || OC create serviceaccount pipeline
```

Comment:
```suggestion
  OC create serviceaccount pipeline 2>/dev/null || true
```

Reply:
same as above, any `oc create` error other than `already exists` should not be ignored